### ISSUE TITLE
[bug] Home directory in jupyterlab is set to /home/jovyan and not /home/qhub_user

### ISSUE THREAD

**Issue by tylerpotts** [Quansight/qhub#559]
## Describe the bug
The home directory for qhub is `/home/jovyan` and not `/home/qhub_user` as expected. This essentially makes qhub unusable, because there aren't sufficient permissions to create files on `/home/jovyan` and there's no way to open notebooks from `/home/qhub_user` 

Example after creating a new terminal session
```bash
tylerpotts@jupyter-tylerpotts$ pwd
/home/jovyan
tylerpotts@jupyter-tylerpotts:/home/jovyan$ touch test.txt
touch: cannot touch 'test.txt': Permission denied
tylerpotts@jupyter-tylerpotts:/home/jovyan$ cd ..
tylerpotts@jupyter-tylerpotts:/home$ ls
conda  jovyan  qhub_user  shared
tylerpotts@jupyter-tylerpotts:/home$ cd qhub_user/
tylerpotts@jupyter-tylerpotts:~$ ls
shared
tylerpotts@jupyter-tylerpotts:~$ touch test.txt
tylerpotts@jupyter-tylerpotts:~$ ls
shared  test.txt
```

## Environment
Deployed on AWS
QHub version `0.3.7`

**Comment by dharhas** [comment]
This happened on another client as well I think @leej3 how did we fix there?

**Comment by leej3** [comment]
We didn't have the issue that our default directory was /home/jovyan. But some personal .bashrc etc. made references to /home/jovyan. That was solved by replacing it with `~` or changing it explicatly to /home/qhub_user.

It's not clear why the notebook server or default working directory would be running from /home/jovyan though. That didn't happen on our deployment.

**Comment by leej3** [comment]
I think this is the file you need to modify for the deployment (by default deployments now have the shell expressly set to /bin/bash) 
https://github.com/Quansight/qhub/commit/7bd70924a15bd673c52bb23e3d8c3dc0dd628be2

**Comment by dharhas** [comment]
per @costrouc changing the default user from `jovyan` to `qhub_user` has had unintended consequences and we is going to revert.

**Comment by costrouc** [comment]
Resolved in release 0.3.10. Reverted back to jovyan

**Comment by costrouc** [comment]
Let me know if the issue still persists
